examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/clientlibs/forms-clientlib-site/styles/components/adaptive-form/accordion.less

```diff
@@ -1,6 +1,6 @@
 .cmp-accordion {
-  .panelContainer();
-  margin-right: @field-right-margin;
+  //.panelContainer();
```

Comment:
why this is commented?
cc @amit-mnnit

Reply:
accordion's css classes are messing up with site's css and breaking the example page